lib/koans/01_equalities.ex

```diff
@@ -1,6 +1,13 @@
 defmodule Equalities do
   use Koans
 
+  @intro  """
```

Comment:
Fixed, though I would probably suggest we improve 'the story' in different PR. 
That line was just an example 😸